test/cluster/test_view_building_coordinator.py

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (C) 2025-present ScyllaDB
+#
+# SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+#
+import asyncio
+import pytest
+import logging
+import time
+from test.pylib.manager_client import ManagerClient
+from test.pylib.util import wait_for
+from test.cluster.util import new_test_keyspace
+
+
+logger = logging.getLogger(__name__)
+
+async def wait_for_view_is_built(cql, ks, view):
+    async def check_table():
+        result = await cql.run_async(f"SELECT * FROM system.built_tablet_views WHERE keyspace_name='{ks}' AND view_name='{view}'")
+        return True if len(result) == 1 else None
+    await wait_for(check_table, time.time() + 60)
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("rf", [1, 2])
+async def test_tablet_view_building(manager: ManagerClient, rf):
+    servers = await manager.servers_add(2)
```

Comment:
Let's start using the `property_file` for new rf>1 tests so that we won't have to adjust it later for rf-valid keyspaces